evm/src/executor/inspector/cheatcodes/mod.rs

```diff
@@ -881,14 +882,15 @@ where
                                 .into_iter()
                                 .flatten()
                                 .join(" and ");
+                                let failure_message = match status {
```

Comment:
Really feeling that we should refactor this file a bit further to get all the inline clutter out into functions—will open an issue for this

Reply:
agree, that would make things a bit nicer